tests/foreman/api/test_host.py

```diff
@@ -298,10 +298,15 @@ def test_positive_create_with_inherited_params(self):
             location=loc,
             organization=org,
         ).create()
-        self.assertEqual(len(host.all_parameters), 2)
+        # get global parameters
+        glob_param_list = {(param.name, param.value) for param
+                           in entities.CommonParameter().search()}
```

Comment:
perhaps we have to assert that len(glob_param_list) > 0 , and if len(glob_param_list) == 0 perhaps we should create some global one what do you think?

Reply:
I don't see why, first in 6.4 there will probably be some global parameters by default, also even without glob parameters still it makes sense to test if host inherits nothing, and third we already have some tests checking global params specifically.

Reply:
How it can inherit some thing that may not exist, there is a probability that the sets will be empty comparing empty sets for me is not interesting, so why we have changed this test.
for me to say that this test cover global parameters we should minimum ensure that global params exits.

Reply:
well, I still don't think it is necessery, reasons in my previous comment. Anyway, for the sake of not blocking the merge, I added what you propse